camayoc/tests/qpc/api/v1/sources/test_sources_common.py

```diff
@@ -413,8 +433,12 @@ def test_type_mismatch(src_type, cleanup, shared_client):
     :expectedresults: An error is thrown and no new source is created.
     :caseautomation: notautomated
     """
+    if src_type == "openshift":
+        pytest.skip("Requires OpenShift")
     src = gen_valid_source(cleanup, src_type, "localhost", create=False)
-    other_types = set(QPC_SOURCE_TYPES).difference(set((src_type,)))
+    qpc_src_types_minus_openshift = set(QPC_SOURCE_TYPES)
```

Comment:
This is related to @bruno-fs previous question, about skipping a test that includes openshift cred. Maybe you could do something like this for creds as well, to make sure no testes are being skiped unintentionally.